site/content/en/docs/Advanced/multi-cluster-allocation.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Multi-cluster Allocation"
+date: 2019-10-25T05:45:05Z
+description: >
+  In order to allow allocation from multiple clusters, Agones provides a mechanism to set redirect rules for allocation requests to the right cluster.
```

Comment:
This page should also make it very clear that this is alpha functionality and subject to change (ideally linking to @markmandel's page about feature stages if that merges in time).

Reply:
Please also add a warning to this page.